bundle/src/bundler.rs

```diff
@@ -65,11 +74,24 @@ impl BundlerUtil {
             })?;
 
         if let Some(CodeOwners { ref path, .. }) = self.meta.base_props.codeowners {
-            let mut file = std::fs::File::open(path)?;
+            let mut file = File::open(path)?;
             tar.append_file("CODEOWNERS", &mut file)?;
             total_bytes_in += std::fs::metadata(path)?.len();
         }
 
+        if let Some(bep_parser) = self.bep_parser.as_ref() {
+            let mut bep_events_file = tempfile::tempfile()?;
+            bep_parser.bep_test_events().iter().for_each(|event| {
+                if let Err(e) = serde_json::to_writer(&bep_events_file, event) {
+                    log::error!("Failed to write BEP event: {}", e);
+                }
+            });
+            bep_events_file.flush()?;
+            bep_events_file.seek(std::io::SeekFrom::Start(0))?;
```

Comment:
This seemed to be necessary in order for the tests to work on Mac.

Reply:
Yeah, I think it's necessary when you write then immediately read again within the same scope